src/main/resources/templates/db_scripts/oaipmh/addCompleteUpdatedDate.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE ${myuniversity}_${mymodule}.instance
+  ADD COLUMN IF NOT EXISTS
+    completeUpdatedDate timestamp with time zone;
```

Comment:
please use snake case for column names

Reply:
Will it be okay if I use completeupdateddate?

Reply:
why not complete_updated_date? any restrictions/limitations?

Reply:
No restrictions, but in this case I need to update code in mod-oai-pmh module. Anyway, no problems, I will use complete_updated_date.